tests/test_handler.py

```diff
@@ -13,26 +14,118 @@ def test_handle(self):
+            f'pysubyt --force  --output ".{os.path.sep}OUTPUT" --templates ".{os.path.sep}TEMPLATE_JINJA_ROOT" --name "TEMPLATE_FILE_NAME" --set "SET_KEY_1" ".{os.path.sep}SET_VALUE_1" --set "SET_KEY_2" ".{os.path.sep}SET_VALUE_2" --var "VARIABLE_KEY_1" "VARIABLE_VALUE_1" --var "VARIABLE_KEY_2" "VARIABLE_VALUE_2" --mode "MODE"',  # noqa
+            f'pysubyt --force --input ".{os.path.sep}INPUT" --output ".{os.path.sep}OUTPUT" --templates ".{os.path.sep}TEMPLATE_JINJA_ROOT" --name "TEMPLATE_FILE_NAME" --mode "iteration"',  # noqa
+            f'pysubyt --force --input ".{os.path.sep}INPUT" --output ".{os.path.sep}OUTPUT" --templates ".{os.path.sep}TEMPLATE_JINJA_ROOT" --name "TEMPLATE_FILE_NAME" --mode "MODE"',  # noqa
+            f'pysubyt --force --input ".{os.path.sep}INPUT" --output ".{os.path.sep}OUTPUT" --templates ".{os.path.sep}TEMPLATE_JINJA_ROOT" --name "TEMPLATE_FILE_NAME" --var "VARIABLE_KEY_1" "VARIABLE_VALUE_1" --var "VARIABLE_KEY_2" "VARIABLE_VALUE_2" --mode "iteration"',  # noqa
+            f'pysubyt --force --input ".{os.path.sep}INPUT" --output ".{os.path.sep}OUTPUT" --templates ".{os.path.sep}TEMPLATE_JINJA_ROOT" --name "TEMPLATE_FILE_NAME" --var "VARIABLE_KEY_1" "VARIABLE_VALUE_1" --var "VARIABLE_KEY_2" "VARIABLE_VALUE_2" --mode "MODE"',  # noqa
+            f'pysubyt --force --input ".{os.path.sep}INPUT" --output ".{os.path.sep}OUTPUT" --templates ".{os.path.sep}TEMPLATE_JINJA_ROOT" --name "TEMPLATE_FILE_NAME" --set "SET_KEY_1" ".{os.path.sep}SET_VALUE_1" --set "SET_KEY_2" ".{os.path.sep}SET_VALUE_2" --mode "iteration"',  # noqa
+            f'pysubyt --force --input ".{os.path.sep}INPUT" --output ".{os.path.sep}OUTPUT" --templates ".{os.path.sep}TEMPLATE_JINJA_ROOT" --name "TEMPLATE_FILE_NAME" --set "SET_KEY_1" ".{os.path.sep}SET_VALUE_1" --set "SET_KEY_2" ".{os.path.sep}SET_VALUE_2" --mode "MODE"',  # noqa
+            f'pysubyt --force --input ".{os.path.sep}INPUT" --output ".{os.path.sep}OUTPUT" --templates ".{os.path.sep}TEMPLATE_JINJA_ROOT" --name "TEMPLATE_FILE_NAME" --set "SET_KEY_1" ".{os.path.sep}SET_VALUE_1" --set "SET_KEY_2" ".{os.path.sep}SET_VALUE_2" --var "VARIABLE_KEY_1" "VARIABLE_VALUE_1" --var "VARIABLE_KEY_2" "VARIABLE_VALUE_2" --mode "iteration"',  # noqa
+            f'pysubyt --force --input ".{os.path.sep}INPUT" --output ".{os.path.sep}OUTPUT" --templates ".{os.path.sep}TEMPLATE_JINJA_ROOT" --name "TEMPLATE_FILE_NAME" --set "SET_KEY_1" ".{os.path.sep}SET_VALUE_1" --set "SET_KEY_2" ".{os.path.sep}SET_VALUE_2" --var "VARIABLE_KEY_1" "VARIABLE_VALUE_1" --var "VARIABLE_KEY_2" "VARIABLE_VALUE_2" --mode "MODE"',  # noqa
         ]
         self.assertEqual(actual, expected)
+
+
+class TestPyshaclHandler(TestCase):
+    def test_handle(self):
+        task = Task(
+            "./tests/resources/input_data",
+            ".",
+            "./tests/resources/sembench_data",
```

Comment:
we should enclosed the paths with pathlib, it will solve different OS issues for instance

Reply:
I can do so as a best practice, although I can't imagine any issues arising here (`/` works both on windows, mac and linux).

Reply:
Please do